test/e2e/playwright/swap/specs/swap.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { test } from '@playwright/test';
+
+import { ChromeExtensionPage } from '../support/extension-page';
+import { SignUpPage } from '../support/signup-page';
+import { NetworkController } from '../support/network-controller-page';
+import { SwapPage } from '../support/swap-page';
+
+let swapPage, networkController;
+
+test.beforeEach('Initialize extension and import wallet', async () => {
```

Comment:
could this be avoided to simplify the test and use the extension-loader.ts setup? Thinking in when you have more tests so you will have less repeated code across specs